website/server/controllers/api-v3/user/spells.js

```diff
@@ -130,7 +130,22 @@ api.castSpell = {
 
       if (party && !spell.silent) {
         let message = `\`${user.profile.name} casts ${spell.text()}${targetType === 'user' ? ` on ${partyMembers.profile.name}` : ' for the party'}.\``;
```

Comment:
It would be awesome if all of these hard-coded messages could use the new localized strings instead, even if the `en` lang is still explicitly used, so these strings don't have to be hunted down and changed every time the localized strings get updated.

ex: website/server/conrollers/api-v3/tasks/groups.js (line 201)